sklearn/feature_selection/tests/test_from_model.py

```diff
@@ -95,12 +208,16 @@ def test_feature_importances():
     assert_almost_equal(importances, importances_bis)
 
     # For the Lasso and related models, the threshold defaults to 1e-5
-    transformer = SelectFromModel(estimator=Lasso(alpha=0.1))
+    transformer = SelectFromModel(estimator=Lasso(alpha=0.1), threshold=1e-5)
```

Comment:
You shouldn't need to change that here. You have to change the meaning of the default, as @jnothman said in his comment.